kvm/requests.yaml

```diff
@@ -8,7 +8,7 @@ releases:
           version: ">= 1.0.1"
           issue: https://github.com/giantswarm/giantswarm/issues/14205
         - name: cluster-operator
-          version: ">= 0.24.2 < 0.25.0"
+          version: ">= 0.24.2"
```

Comment:
Upgrading app-operator too so we don't need this limitation anymore: https://github.com/giantswarm/releases/pull/650#discussion_r597052466